pom.xml

```diff
@@ -13,8 +13,8 @@
     <!-- 17-May-2023, tatu: [json-api#172] Need at least Jackson 2.15.x -->
     <!-- 16-Nov-2023, tatu: but may use 2.16 now that it's available -->
     <jackson.version>2.16.2</jackson.version>
-    <!-- 09-Jul-2024, tatu: temporary override here until SG 2.0.28 -->
-    <quarkus.platform.version>3.9.5</quarkus.platform.version>
+    <!-- 14-Aug-2024, tatu: override here until Stargate gets to this version too -->
+    <quarkus.platform.version>3.13.2</quarkus.platform.version>
```

Comment:
I thought we had fully severed ourselves from Stargate pom? Why can't we do that?

Reply:
maybe @Yuqi-Du knows?

Reply:
We should, definitely. But haven't yet I think. I can change the comment to reflect this.

Reply:
actually, probably fine as is. We need to define quarkus version even afterwards.

Reply:
How about I'll create GH issue for removal.

Reply:
-> #1341